catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlDatabaseOperations.java

```diff
@@ -85,6 +85,24 @@ public String generateDropDatabaseSql(String databaseName, boolean cascade) {
     return dropDatabaseSql;
   }
 
+  @Override
+  protected void validateDatabaseName(String databaseName) {
+    if (StringUtils.isBlank(databaseName)) {
+      throw new IllegalArgumentException("Database name cannot be empty.");
+    }
+    if (databaseName.length() > 64) {
+      throw new IllegalArgumentException("Database name cannot be longer than 64 characters.");
+    }
+    // Regex that matches any string that consists entirely of word characters, Unicode letters, or
+    // dollar signs
+    // \w matches [a-zA-Z0-9_]
+    // \p{L} matches any kind of letter from any language
+    if (!databaseName.matches("^[\\w\\p{L}$]*$")) {
```

Comment:
Is this a rule from experience? Or is it an official MySQL constraint?

Reply:
Oh, I just saw the description in your PR, it's a limitation of MySQL official.

Reply:
Yes, this is the most basic permitted characters in unquoted identifiers for MySQL. I think most DB users uses only these characters. But other characters can also be used with quoted identifiers. Do you think we should include those characters as well?

Reply:
I think we can start with stricter restrictions and gradually relax them as needed.